packages/actions/test/e2e/00-authentication.test.ts

```diff
@@ -56,24 +62,47 @@ describe("Authentication", () => {
         )
     })
 
-    it("should not be possible to authenticate twice", async () => {})
+    it("should not be possible to authenticate if the user has been disabled from the Authentication service by coordinator", async () => {
+        // Disable user.
+        const disabledRecord = await adminAuth.updateUser(user.uid, { disabled: true })
+        expect(disabledRecord.disabled).to.be.true
 
-    it("should not be possible to authenticate if the user refuses to associate its Github account", async () => {})
+        await expect(signInWithEmailAndPassword(userAuth, user.data.email, userPassword)).to.be.rejectedWith(
+            "Firebase: Error (auth/user-disabled)."
+        )
 
-    it("should not be possible to authenticate if the user send an expired device token", async () => {})
+        // re enable the user
+        const recordReset = await adminAuth.updateUser(user.uid, {
+            disabled: false
+        })
+        expect(recordReset.disabled).to.be.false
+    })
 
-    it("should not be possible to authenticate if Github is unreachable", async () => {})
+    it("should not be possible to authenticate with an incorrect password", async () => {
+        await expect(signInWithEmailAndPassword(userAuth, user.data.email, "wrongPassword")).to.be.rejectedWith(
+            "Firebase: Error (auth/wrong-password)."
+        )
+    })
 
-    it("should not be possible to authenticate if Firebase is unreachable", async () => {})
+    it("should not be possible to authenticate with an incorrect email", async () => {
+        await expect(signInWithEmailAndPassword(userAuth, "wrongEmail", userPassword)).to.be.rejected
+    })
 
-    it("should not be possible to authenticate if the user has been disabled from the Authentication service by coordinator", async () => {})
+    it("should not be possible to authenticate if Firebase is unreachable", async () => {
+        // @todo mock unreachable firebase.
+    })
+    it("should not be possible to authenticate twice", async () => {
+        // @todo Implement checks to prevent double authentication.
```

Comment:
I think double authentication is currently avoided in the `auth` command (by checking local access token). Hmm, idk if there's a better way using Firebase methods (e.g., you got recognize if already auth and try to do it twice).

Reply:
```js
await signInWithEmailAndPassword(userAuth, user.data.email, userPassword)
await signInWithEmailAndPassword(userAuth, user.data.email, userPassword)
```

Authenticating twice with email and password does not give any error. This test might have to be removed from the actions package and be part of possible tests for the cli itself.

Reply:
Cool, thanks for finding this out. So, if aren't any errors from Firebase methods, we could assume to move to the CLI (where is already covered).